rb/lib/selenium/webdriver/common/credential.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#
+# A credential stored in a virtual authenticator.
+# @see https://w3c.github.io/webauthn/#credential-parameters
+#
+
+module Selenium
+  module WebDriver
+    class Credential
+      def initialize(*args)
+        @id = args[0]
+        @is_resident_credential = args[1]
+        @rp_id = args[2]
+        @user_handle = args[3]
+        @private_key = args[4]
+        @sign_count = args[5]
+      end
+
+      attr_reader :id, :is_resident_credential, :rp_id, :private_key, :sign_count
+
+      def user_handle
+        return @user_handle unless @user_handle.nil?
+
+        nil
```

Comment:
```suggestion
        @user_handle
```

This is essentially the same code.

Reply:
Changes done.